data/reference

```diff
@@ -0,0 +1 @@
+/Users/mojmir/projects/etl/data/garden/reference
```

Comment:
Is this file supposed to be here?

Reply:
That's a symlink to the new location with the data. I thought it would be safer to keep it there just in case, but perhaps I should just get rid of it and make anyone who could be using this update it? (I don't think there's anyone)

Reply:
But it seems to be recorded as an absolute path, right? I.e. this won't work on anybody else's machine. Or am I missing something?

Reply:
Ouch... you're right, I totally missed this.